python/src/main/api.py

```diff
@@ -35,7 +36,18 @@ def put(self, email, first_name, last_name):
 @api.route('/api/v1/person/<string:email>')
 class People(Resource):
     def delete(self, email):
-        return 200
+        conn = sqlite3.connect("person_and_item.db")
+        cursor = conn.cursor()
+        db = DB(cursor, conn)
+        person = db.get_person(email)
+        if len(person) == 0:
+            logging.error("Couldn't find person with email '%s'" % email)
+            abort(404)
+        else:
+            db.delete_person(email)
+            logging.info("User with email id %s has been deleted" % email)
+            return 410
```

Comment:
lol, nice cat!

The 400 codes are for errors. 

For a delete you are looking at 200 - okay or 202 - no content

Reply:
Okay, so is it okay to return nothing in these situations?

Reply:
yeah